datadog_checks_tests_helper/datadog_test_libs/win/fixtures/README.md

```diff
@@ -0,0 +1,75 @@
+These fixtures were generated from a Windows server running many services like IIS, Active Directory, etc.
+
+`counter_indexes_<lang>-<locale>` is essentially the contents of the registry entry that lists all the strings, enumerated in the file.
+`allcounters_<lang>-<locale>` is the output of `typeperf -qx`, with a bit of parsing to make it easier to parse back in for the tests.
+
+You take `typeperf -qx` and redirect to a file, that gets you a huge file in the format:
+
+```
+\Storage QoS Filter - Volume(D:)\Avg. Normalized I/O Cost
+\Storage QoS Filter - Volume(D:)\Avg. I/O Cost
+\Network Virtualization(Provider Routing Domain)\Unicast Replicated Packets out
+\Network Virtualization(Provider Routing Domain)\Inbound Packets dropped
+```
+
+Then you use a giant regex to format it so that `!` is the separator, to make it easier to parse.
```

Comment:
The example text above doesn't have any `!` included ..  

Should some of the lines look like this?
```
Storage Spaces Tier ! ! Avg. Tier Bytes/Transfer
Storage Spaces Tier ! ! Tier Transfer Bytes/sec
```

Reply:
in `allcounters_*`

Reply:
Okay, I see the pattern now in that file, the `!` are field delimiters, and in some cases the field is empty, like in my example above.

For better clarity though, it would help if you included the full text of the line showing the `!` so its clear what will be extracted and then parsed.

Reply:
Done!

Reply:
Nice!